design/mappings/file_manifest_mappings.yml

```diff
@@ -0,0 +1,16 @@
+fileManifest:
+  Filename: asset.path
+  Filetype: [
+    asset.path, #Split and get file extension
+    asset.encodingFormat, #Follows application/<filetype> format but type is just string
+    dandiset.metadata.assetsSummary.dataStandard # has RRID + standard name but is at data collection level
```

Comment:
a data standard is not necessarily applicable at the file level. for example, with BIDS at the file level the data standards are NIFTI/JSON/etc.,., will BIDS is a collection level standard.

however, for NWB, indeed it's both an encoding format and a standard. we are trying to figure out more complex MIME types, such `application/vnd+hdf5+nwb` but formal use of such things is still under discussion.

Reply:
That makes sense - removed dataStandard from this mapping. Mime type seems like the sensible field for this.